apps/req.c

```diff
@@ -44,6 +44,17 @@
 #define DEFAULT_KEY_LENGTH      2048
 #define MIN_KEY_LENGTH          512
 
+/* Is this a separator? Always in ASCII */
+#define ASCSEPARATOR(c) (c == ':' || c == ',' || c == '.')
```

Comment:
As it is a macro :
+#define ASCSEPARATOR(c) ((c) == ':' || (c) == ',' || (c) == '.')